### PR TITLE
[ADDED]  FAQ Section

### LINKED ISSUE #7: [NEW] FAQ Section
image

1. maintain padding and responsiveness in accordion
2. do not put gradient this section which is extended in the upper component

### PR DESCRIPTION
**Pull request by amitsuthar69**
### Description :
Added FAQ Section.

---

### Fixes :
Fixes #7.

---

### Checklist :
- [x] Added FAQSection folder in components folder in src directory.
- [x] Created FAQSection.jsx and faq-data.js inside FAQSection folder.
- [x] Created FAQSection folder in public directory and added an image in svg format.
- [x] Did not hard code the data. Mapped data from faq-data.js to FAQSection.jsx file.
- [x] Added FAQSection component to App.jsx file.
- [x] Maintained proper spacings and paddings.
- [x] Added responsiveness.

---

### Feature :

**Desktop View :**
image

---

**Tablet View :**
image

---

**Mobile View :**
image

### CODE REVIEW

**Comment by Rahilsiddique** [comment]
LGTM

**Review by Rahilsiddique — APPROVED** [review]

File: src/components/FAQSection/FAQSection.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import faqData from "./faq-data";
+
+const FAQSection = () => {
+  return (
+    <section className="px-8 my-12 text-lg font-inter lg:p-16">
```
Comment (kamalika0363) [inline src/components/FAQSection/FAQSection.jsx:6]: <section className="px-8 my-12 lg:p-16">

Comment (kamalika0363) [reply src/components/FAQSection/FAQSection.jsx:6]: <section className="px-8 my-12 lg:p-16">

Comment (kamalika0363) [reply src/components/FAQSection/FAQSection.jsx:6]: <section className="px-8 my-12 lg:p-16">

File: src/components/FAQSection/FAQSection.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="flex flex-col gap-2 mx-auto md:px-16 md:py-14 lg:w-3/4">
+        <h1 className="text-3xl font-medium text-white">FAQ</h1>
+        {faqData.map(({ id, question, answer }) => (
+          <details
```
Comment (kamalika0363) [inline src/components/FAQSection/FAQSection.jsx:10]: <details
            key={id}
            className="rounded-xl cursor-pointer group border-1 border-solid border-[#170F1E]"
            style={{
              background:
                "linear-gradient(145deg, rgba(75, 72, 72, 0.30) 13.28%, rgba(187, 180, 180, 0.21) 87.52%)",
            }}
          >

File: src/components/FAQSection/FAQSection.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            key={id}
+            className="p-1 bg-gradient-to-tr rounded-xl border border-gray-800 cursor-pointer group from-slate-800 to-slate-900"
+          >
+            <summary className="flex gap-2 justify-between items-center p-3 text-sm text-purple-300">
```
Comment (kamalika0363) [inline src/components/FAQSection/FAQSection.jsx:14]: <summary className="flex gap-2 justify-between items-center px-4 md:px-8 py-5 text-lg text-indigo-300 ">

File: src/components/FAQSection/FAQSection.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+              {question}
+              <img
+                src="./FAQSection/arrow.svg"
+                alt="down-arrow"
+                className="w-4 h-4 group-open:rotate-180"
+              />
+            </summary>
+            <div className="px-3 mb-4">
```
Comment (kamalika0363) [inline src/components/FAQSection/FAQSection.jsx:22]: <div className="px-4 md:px-8 mb-4">
              <p className="box-content text-lg text-[#AAACCD]">{answer}</p>
            </div>

File: src/components/FAQSection/FAQSection.jsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import faqData from "./faq-data";
+
+const FAQSection = () => {
+  return (
+    <section className="px-8 md:my-4 my-12 font-inter lg:p-16">
```
Comment (kamalika0363) [inline src/components/FAQSection/FAQSection.jsx:6]: px-8 md:my-4 my-12 lg:p-16

**Pull request merged**